bin/ci/semver.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function grep_word_from_git_commits_since_last_tag() {
+    local commits=$(git log --oneline $(git describe --tags --abbrev=0 @^)..@ --grep="$1" 2> /dev/null)
```

Comment:
```suggestion
    local commits
    commits=$(git log --oneline "$(git describe --tags --abbrev=0 @^)"..@ --grep="$1" 2> /dev/null)
```

https://github.com/koalaman/shellcheck/wiki/SC2155#problematic-code-in-the-case-of-local

Reply:
Fixed, thanks.